Rakefile

```diff
@@ -17,8 +17,12 @@ namespace :version do
   end
 end
 
-require 'rake/extensiontask'
-Rake::ExtensionTask.new("fiddle")
-Rake::ExtensionTask.new("-test-/memory_view")
+if RUBY_ENGINE == 'ruby'
+  require 'rake/extensiontask'
+  Rake::ExtensionTask.new("fiddle")
+  Rake::ExtensionTask.new("-test-/memory_view")
+else
+  task :compile
+end
```

Comment:
Can we revert this?
We don't need to use `rake default` with TruffleRuby.

Reply:
On one side you seem to be asking a clean integration by upstraming the Fiddle TruffleRuby here soon, and on the other side you seem to dislike any change making incremental progress towards that.
I don't see any point to revert this, it's useful to run fiddle tests on TruffleRuby.

Reply:
I just want to keep 1 logical change in 1 PR. Is it strange?
I think that this PR focus on "workaround for bundled gem" not "upstreaming TruffleRuby implementation".

> the changes in `test/run.rb` is similar, it makes it easier to run tests in order to make them pass

It's needless when we don't use `$LOAD_PATH.delete` approach.